### DISCUSSION TITLE
Vaultwarden with static ip

### DISCUSSION THREAD

**Discussion by Shootify** [tteck/Proxmox#1606] [Help Needed] [answered]
Hello, i have a vps exposed on the internet, how to get vaultwarden script to prompt to enter static ip information? thanks

**Comment by tteck** [comment, marked as answer]
Use "Advanced" settings

**Comment by Shootify** [comment]
great. thanks

**Comment by Shootify** [comment]
should i use privileged or unprivileged container?

**Reply by tteck** [reply]
I would suggest unprivileged.

**Reply by Shootify** [reply]
thanks

**Reply by Shootify** [reply]
Hi for some reason, when using advanced, it stops saying no internet, even though I entered the Public IPV4 CIDR and gateway, am i missing something here?

**Reply by tteck** [reply]
> am i missing something here?

It's important to note that the IP you provided is a local IP within your network, not your public IP.

**Reply by Shootify** [reply]
yes, its local ip provided by my datacenter where i have the server on

**Reply by tteck** [reply]
> provided by my datacenter where i have the server on

From my perspective, that doesn't align with self-hosting.

**Reply by tteck** [reply]
I kindly choose to conclude my participation in the conversation at this time.

**Reply by Shootify** [reply]
That is my server with proxmox, I am renting the rack and IP Stack. I respect if you no longer want to participate in this conversion thought. thanks anyways. 
cya

**Reply by tteck** [reply]
Without direct physical access to the hardware, the hosting cannot be classified as self-hosting. Furthermore, it is unlikely that these scripts will be of much benefit to you in this context.

**Comment by Shootify** [comment]
i dont have a for example reverse proxy at my office, what about if i deploy vaultwarden and use cloudflare tunnel to reach from outside? feedback is appreciated it. thanks a lot.

**Comment by Shootify** [comment]
I managed to install it on my server located on the datastore, now i need to request lets encrypt... can you share best practice. thanks a lot.
